SAT/SATSolver.hpp

```diff
@@ -134,6 +134,8 @@ class SATSolver {
   
   virtual void suggestPolarity(unsigned var, unsigned pol) = 0;
 
+  virtual void requestRandomization(unsigned seed) {}
```

Comment:
Maybe a pure-virtual here isn't so bad? `suggestPolarity` is pure virtual, for example.